packages/react-instantsearch-dom/src/widgets/Answers.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect, useMemo, useContext } from 'react';
+import { instantSearchContext } from 'react-instantsearch-core';
+import { createConcurrentSafePromise } from '../lib/createConcurrentSafePromise';
+import { debounce } from '../lib/debounce';
+
+/* eslint-disable react/prop-types */
+function DefaultAnswersComponent({ isLoading, hits = [] }) {
+  return isLoading ? (
+    <p>loading...</p>
+  ) : (
+    hits && hits.length > 0 && <pre>{JSON.stringify(hits, null, 2)}</pre>
+  );
+}
+
+export default function Answers({
+  searchClient,
+  attributesForPrediction = ['*'],
+  queryLanguages = ['en'],
```

Comment:
@eunjae-lee I suspect this may have come from the documentation? If so, the docs were incorrect on this point. We want `queryLanguages` to be set each time and not default to English.

Reply:
@dustincoates oh I see.
I got it from https://www.algolia.com/doc/rest-api/answers/#method-param-querylanguages
What about `attributesForPrediction = ['*']`. Is it correct?

Reply:
@eunjae-lee we want it to be set to a specific language if at all possible